courses/ahybrid/v1.0/AHYBRID080/scripts/install.sh

```diff
@@ -17,6 +17,7 @@
 source ./scripts/env.sh
 
 apt-get install kubectl
+apt-get install google-cloud-sdk-kpt
```

Comment:
Do we use kpt? Are we doing this because we're matching instructions or we truly require the tool for this?